### PR TITLE
Rename wasi_llm -> wasmedge_llmc

### PR DESCRIPTION
**Pull request by junaire**
Depends on https://github.com/WasmEdge/WasmEdge/pull/3731

### CODE REVIEW

File: wasmedge-llmc/src/llmc_interface.rs [hunk 1]
```diff
@@ -134,14 +134,14 @@ pub unsafe fn model_train(
...
-        Err(WasmedgeLLMErrno(result as u32))
+        Err(WasmedgeLLMCErrno(result as u32))
     } else {
         Ok(())
     }
 }
 
 pub mod wasmedge_llm {
-    #[link(wasm_import_module = "wasi_llm")]
+    #[link(wasm_import_module = "wasmedge_llm")]
```
Comment (hydai) [inline wasmedge-llmc/src/llmc_interface.rs]: This should be `wasmedge_llmc`.

**Review by hydai — CHANGES_REQUESTED** [review]

File: wasmedge-llmc/src/llmc_interface.rs (hunk 1 above)
Comment (junaire) [reply wasmedge-llmc/src/llmc_interface.rs]: good catch, fixed

**Review by hydai — APPROVED** [review]

**Comment by hydai** [comment]
Hi @junaire 
I just merged a PR to disable the WasmEdge AOT feature and remove the `lld` related dependencies to reduce the CI cost. Please fix the conflict in `build.yml`. Thanks.

**Comment by junaire** [comment]
> Hi @junaire I just merged a PR to disable the WasmEdge AOT feature and remove the `lld` related dependencies to reduce the CI cost. Please fix the conflict in `build.yml`. Thanks.

done

**Pull request merged**